html/components/tooltip.js

```diff
@@ -0,0 +1,118 @@
+import getElements from '../utilities/getElements';
+import {
+  isEscPressed,
+  isEnterPressed,
+  isSpacePressed,
+} from '../utilities/keypress';
+import toggleAriaExpanded from '../utilities/toggleAriaExpanded';
+
+/**
+ *  Tooltip JS
+ */
+
+function calculatePositionClass(trigger){
```

Comment:
Some spacing tidying up here and lets write these functions using arrow functions.